lib/kdump_utils.pm

```diff
@@ -365,15 +366,15 @@ sub check_function {
         power_action('reboot', textmode => 1, observe => 1, keepconsole => 1);
     }
     unlock_if_encrypted;
-    # Wait for system's reboot; more time for Hyper-V as it's slow.
-    $self->wait_boot(bootloader_time => check_var('VIRSH_VMM_FAMILY', 'hyperv') ? 200 : undef);
+    # Wait for system's reboot; more time for Hyper-V / aarch64 as it's slow.
+    $self->wait_boot(bootloader_time => $boot_timeout);
     select_console 'root-console';
 
     assert_script_run 'find /var/crash/';
 
     if ($args{test_type} eq 'function') {
         # Check, that vmcore exists, otherwise fail
-        assert_script_run('ls -lah /var/crash/*/vmcore', 180);
+        assert_script_run('ls -lah /var/crash/*/vmcore', is_aarch64 ? 240 : 180);
```

Comment:
In principle ok, but why not simply increase the timeout to 240 everywhere?

Pro: It's simpler and nobody will ask in a year: "why the heck is this different on aarch64?"  
Cons: We might loose 60 seconds of test time in a very rare corner case where the test run is anyways doomed.

```suggestion
        assert_script_run('ls -lah /var/crash/*/vmcore', 240);
```